doc/articles/guides/uno-material-walkthrough.md

```diff
@@ -98,23 +98,19 @@ This guide will walk you through the necessary steps to setup and to use the [`U
                 <!-- Load WinUI resources -->
                 <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
+                <!-- Load Material Color Palette -->
```

Comment:
What about fonts?

Reply:
I feel like fonts are all messed up at this point. I need to do more research on how to get fonts included from a library and how to override those if needed. Issue incoming